### ISSUE TITLE
no clear way to copy configuration from windows

### ISSUE THREAD

**Issue by nster101** [andreasomaini/apex-m800-cfg#1]
Hi, I am not sue what kind of files are required when attempting to copy a windows configuration. Could you clarify this?

**Comment by andreasomaini** [comment]
Yeah you are right, I actually forgot to ever update the instructions on the README 

You have to sniff the USB packets sent from the official SteelSeries Engine, put the binary values in a file, then you can upload that file to the keyboard using this software

I’ll provide clearer instructions tomorrow anyway if you didn’t understand

**Comment by andreasomaini** [comment]
@nster101 
I added in depth instruction on the README with the last commit ( 88e0c74fbe3561be639e09787991d9aad3a58379 )

Please let me know if they are clear enough for you!
